### PR TITLE
Delete resources after 7 days, send mail after 4 days of creation

### PR DESCRIPTION
**Pull request by athiruma**
1. Added skip resource deletion.
2. Trigger mail after 4 days of  resource
3. Delete ebs after 7 days of unused resource
#### Deleted Resources
- ebs_unattached
- zombie_elastic_ips
- zombie_nat_gateways
- empty_roles
- empty_buckets
- zombie_snapshots

### CODE REVIEW

File: cloud_governance/common/clouds/aws/cloudtrail/cloudtrail_operations.py [hunk 1]
```diff
@@ -5,10 +5,13 @@
 
 import boto3
 
+from cloud_governance.common.logger.init_logger import logger
+
 
 class CloudTrailOperations:
     SEARCH_SECONDS = 10
     SLEEP_SECONDS = 120
+    DAYS_GO_BACK = 30
```
Comment (ebattat) [inline cloud_governance/common/clouds/aws/cloudtrail/cloudtrail_operations.py:14]: What it means?

Comment (athiruma) [reply cloud_governance/common/clouds/aws/cloudtrail/cloudtrail_operations.py:14]: If I didn't find a username in the tags, I search in the cloud trail. Some resources don't have create time. So I search for the last 30 days

Comment (ebattat) [reply cloud_governance/common/clouds/aws/cloudtrail/cloudtrail_operations.py:14]: Why 30 and not 90 days ?

Comment (athiruma) [reply cloud_governance/common/clouds/aws/cloudtrail/cloudtrail_operations.py:14]: I thought 30 days is large enough.

File: cloud_governance/common/clouds/aws/cloudtrail/cloudtrail_operations.py [hunk 2]
```diff
@@ -147,24 +173,29 @@ def __get_time_difference(self, start_time: datetime):
...
-        username, event = self.__get_user_by_resource_id(start_time, end_time, resource_id, resource_type)
+        if start_time:
+            delay_seconds = int(os.environ.get('SLEEP_SECONDS', self.SLEEP_SECONDS))
+            if self.__get_time_difference(start_time=start_time) <= delay_seconds:
+                time.sleep(delay_seconds)
+            search_time = timedelta(seconds=self.SEARCH_SECONDS)
+            end_time = start_time + search_time
+            start_time = start_time - search_time
+        else:
+            start_time = datetime.now() - timedelta(days=self.DAYS_GO_BACK)
```
Comment (ebattat) [inline cloud_governance/common/clouds/aws/cloudtrail/cloudtrail_operations.py:196]: DAYS_GO_BACK => LOOKBACK_DAYS

File: cloud_governance/aws/zombie_non_cluster/run_zombie_non_cluster_policies.py [hunk 3]
```diff
@@ -125,5 +137,134 @@ def _get_policy_value(self, tags: list):
...
+        """
+        This method get last used day from the tags
+        @param tags:
+        @return:
+        """
+        last_used_days = self._get_tag_name_from_tags(tags=tags, tag_name='LastUsedDay')
+        if not last_used_days:
+            last_used_days = 1
+        else:
+            last_used_days = int(last_used_days) + 1
```
[the remark this answers is not in the document]
Comment (athiruma) [reply cloud_governance/aws/zombie_non_cluster/run_zombie_non_cluster_policies.py]: resolved

File: cloud_governance/common/clouds/aws/cloudtrail/cloudtrail_operations.py (hunk 2 above)
Comment (athiruma) [reply cloud_governance/common/clouds/aws/cloudtrail/cloudtrail_operations.py]: resolved

**Review by ebattat — APPROVED** [review]
/LGTM

**Pull request merged**